model-server/src/main/kotlin/org/modelix/model/server/handlers/ModelReplicationServer.kt

```diff
@@ -165,6 +165,14 @@ class ModelReplicationServer(val repositoriesManager: RepositoriesManager) {
             call.respondDelta(initialVersion.getContentHash(), null)
         }
 
+        post<Paths.deleteRepository> {
+            fun ApplicationCall.repositoryId() = RepositoryId(parameters["repository"]!!)
+            fun PipelineContext<Unit, ApplicationCall>.repositoryId() = call.repositoryId()
+
+            repositoriesManager.removeRepository(repositoryId())
+            call.respond(HttpStatusCode.OK)
```

Comment:
HTTP No Content would probably be the more appropriate response code to use here.

Reply:
fixed in [81d8615](https://github.com/modelix/modelix.core/pull/450/commits/81d8615efaf13c61e4af2ee6dc3a83ec979293a5)